bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/NoResultTests.kt

```diff
@@ -0,0 +1,91 @@
+package net.gini.android.bank.sdk.exampleapp.ui.testcases
+
+import android.Manifest
+import androidx.test.espresso.IdlingRegistry
+import androidx.test.ext.junit.rules.activityScenarioRule
+import androidx.test.rule.GrantPermissionRule
+import net.gini.android.bank.sdk.exampleapp.ui.MainActivity
+import net.gini.android.bank.sdk.exampleapp.ui.resources.ImageUploader
+import net.gini.android.bank.sdk.exampleapp.ui.resources.PdfUploader
+import net.gini.android.bank.sdk.exampleapp.ui.resources.SimpleIdlingResource
+import net.gini.android.bank.sdk.exampleapp.ui.screens.CaptureScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.MainScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.NoResultScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.OnboardingScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.ReviewScreen
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+/**
+ * Test class for No Result screen.
+ */
+class NoResultScreenTests {
+
+    @get:Rule
+    val activityRule = activityScenarioRule<MainActivity>()
+
+    @get: Rule
+    val grantPermissionRule: GrantPermissionRule =
+        GrantPermissionRule.grant(Manifest.permission.CAMERA)
+
+    private val mainScreen = MainScreen()
+    private val onboardingScreen = OnboardingScreen()
+    private val captureScreen = CaptureScreen()
+    private val imageUploader = ImageUploader()
+    private val pdfUploader = PdfUploader()
+    private val reviewScreen = ReviewScreen()
+    private val noResultScreen = NoResultScreen()
+    private lateinit var idlingResource: SimpleIdlingResource
+
+    @Before
+    fun setup() {
+        idlingResource = SimpleIdlingResource(5000)
+        IdlingRegistry.getInstance().register(idlingResource)
+    }
+
+    @Test
+    fun test1_uploadInvalidImageAndClickEnterManuallyButton_NavigatesToMainScreen() {
```

Comment:
[Request] Please update this method to make an assertion at the end, like you did in [this](https://github.com/gini/gini-mobile-android/pull/565) PR.